tests/dist/python/run_dist_objects.py

```diff
@@ -0,0 +1,88 @@
+import dgl
+import torch
+import os
+import argparse
+import numpy as np
+import dgl.backend as F
+from dgl.distributed import partition_graph, load_partition, load_partition_book
+import time
+
+args_parser = argparse.ArgumentParser()
```

Comment:
Environment variables prefixed with `DIST_DGL_TEST_` are used for passing in `rpc_basics.py`. Args is used here. I think both are ok. But could we use a same/unified passing logic? some of the args are required in many unit tests such as `client/server`, `workspace`, `net_type`. The reason why I chose env for passing is mainly env is highly used within DistDGL.